specutils/manipulation/extract_spectral_region.py

```diff
@@ -161,3 +161,36 @@ def extract_region(spectrum, region):
         extracted_spectrum = extracted_spectrum[0]
 
     return extracted_spectrum
+
+
+def spectral_slab(spectrum, s1, s2):
+    """
+    Extract a slab from the input `~specutils.Spectrum1D`
+    defined by the lower and upper bounds defined by the ``region``
+    instance.  The extracted region will be returned as a new
+    `~specutils.Spectrum1D`.
+
+    Parameters
+    ----------
+    spectrum: `~specutils.Spectrum1D`
+        The spectrum object from which the region will be extracted.
+
+    s1, s2: spectral coordinates limiting the slab
+        Define the spectral coordinate bound of the region to extract
+        from the original spectrum.
+
+    Returns
+    -------
+    spectrum: `~specutils.Spectrum1D` or list of `~specutils.Spectrum1D`
+        Excised spectrum, or list of spectra if the input region contained multiple
+        subregions.
+
+    Notes
+    -----
+    This is for now just a proxy for function `extract_region`, to ease the
+    transition from `spectral-cube`.
```

Comment:
It looks like this line is causing the tests to fail, with the quotes the docs are expecting `spectral-cube` to reference an object defined in this file (I think).
```suggestion
    transition from the spectral-cube package.
```